pkg/kubectl/cmd/util/helpers.go

```diff
@@ -270,7 +270,7 @@ func GetFlagDuration(cmd *cobra.Command, flag string) time.Duration {
 
 func AddValidateFlags(cmd *cobra.Command) {
 	cmd.Flags().Bool("validate", true, "If true, use a schema to validate the input before sending it")
-	cmd.Flags().String("schema-cache-dir", "/tmp/kubectl.schema", "If non-empty, load/store cached API schemas in this directory, default is '/tmp/kubectl.schema'")
+	cmd.Flags().String("schema-cache-dir", "~/.kube/schema", "If non-empty, load/store cached API schemas in this directory, default is '~/kubectl.schema', where '~' is replaced with $HOME by kubectl")
```

Comment:
As mentioned above, recommend using a var initialized in `clientcmd` as the default value and in the help text so there's only one place to keep it updated. Case in point, the help text here doesn't match the default value.